src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlInstanceTransparentDataEncryptionCertificate.md

```diff
@@ -1,25 +1,25 @@
-﻿---
-external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
-Module Name: Az.Sql
-online version: https://docs.microsoft.com/en-us/powershell/module/az.sql/Add-AzSqlManagedInstanceTransparentDataEncryptionCertificate
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
```

Comment:
Please use this instead:
external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
Module Name: Az.Sql
online version: https://docs.microsoft.com/en-us/powershell/module/az.sql/Add-AzSqlInstanceTransparentDataEncryptionCertificate

Apply everywhere

Reply:
Ok, thanks. I actually have to remove this file (Add-AzureRmSqlInstanceTransparentDataEncryptionCertificate.md) since I already have Add-AzSqlInstanceTransparentDataEncryptionCertificate.md which has correct values